js/index.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable import/extensions */
+
+import { keysDataEn, keysDataRu } from './data.js';
+import Keyboard from './modules/Keyboard.js';
+import * as helper from './modules/helper.js';
+
+window.onload = () => {
+  const keyboard = new Keyboard();
+
+  keyboard.createKeyboard();
+
+  const insertCurrentLanguage = (data) => {
+    const { lang } = data;
+    if (lang === 'en') {
+      keyboard.paintKeyboard(keysDataEn);
+      helper.pushDataInLocalStorage(keyboard.lang = 'en');
+    }
+    if (lang === 'ru') {
+      keyboard.paintKeyboard(keysDataRu);
+      helper.pushDataInLocalStorage(keyboard.lang = 'ru');
+    }
+  };
+
+  const value = helper.loadDataFromLocalStorage();
+  if (value) {
+    insertCurrentLanguage(value);
```

Comment:
If value is a language, that it would be better to give this method of helper and for variable `value` more descriptive names.